Misc/NEWS.d/next/Library/2022-10-08-19-39-27.gh-issue-98086.y---WC.rst

```diff
@@ -0,0 +1 @@
+Make sure ``patch.dict()`` can be applied on async functions.
```

Comment:
For future PRs, please make sure that the news fragment makes sense when it’s read in the compiled changelog.
Here `patch` is mentioned but there is no such builtin, it would be better to name `unittest.mock.patch.dict` (like the issue helpfully did).  Thanks!